sunpy/map/header_helper.py

```diff
@@ -405,6 +405,8 @@ def make_heliographic_header(date, observer_coordinate, shape, *, frame, project
         Coordinate frame.
     projection_code : {'CAR', 'CEA'}
         Projection to use for the latitude coordinate.
+    map_center_longitude : `astropy.units.Quantity` (equivalent units to `u.deg`)
+        Heliographic longitude of the center pixel
```

Comment:
```suggestion
    map_center_longitude : `~astropy.units.Quantity`
        Heliographic longitude of the map center
```
There is a "center pixel" only if the map has an odd number of rows and columns.

Reply:
Got it, thanks